app/src/main/java/com/example/people4animals/LogInView.kt

```diff
@@ -33,9 +33,9 @@ class LogInView : AppCompatActivity() {
 
     private fun login(view: View) {
 
-        if (binding.loginEmailETChild.text.toString() != "" && binding.loginPassETChild.text.toString() != "") {
-            val email = binding.loginEmailETChild.text.toString()
-            val password = binding.loginPassETChild.text.toString()
+        if(binding.loginEmailETChild.editText!!.text.toString() != "" && binding.loginPassETChild.editText!!.text.toString() != "" ){
```

Comment:
Realizalo con un let

Reply:
This is a correct way to do it, because in this moment we have security of this is a valid value